build.sbt

```diff
@@ -46,7 +46,7 @@ val mavenSettings = Seq(
 
 val commonSettings = Seq(
   scalaVersion := "2.12.3",
-  crossPaths := false,
+  crossScalaVersions := Seq("2.11.8", scalaVersion.value),
```

Comment:
`2.11.11`?

Reply:
it should be! This PR is just reverting the previous merge though

Reply:
Fair enough!